### PR TITLE
nbgrader version 0.5.0

### LINKED ISSUE #1: Test against conda-forge phantomjs, fontconfig, freetype
Once [PhantomJS is feedstocked](https://github.com/conda-forge/staged-recipes/pull/692), we should use it and get rid of the yum_requirements!

See https://github.com/conda-forge/nbpresent-feedstock/issues/1

### PR DESCRIPTION
**Pull request by jhamrick**
This updates the feedstock for nbgrader version 0.5.0 and also fixes #1

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,14 +11,9 @@ source:
...
   sha256: {{ sha256 }}
-  patches:
-    - PR695.patch
-    - PR700.patch
-    - PR701.patch
-    - PR704.patch
 
 build:
-  number: 0
+  number: 1
```
Comment (lgpage) [inline recipe/meta.yaml]: build number should be 0

**Review by lgpage — CHANGES_REQUESTED** [review]
please could you also re-render this with conda smithy 2.3.2

File: recipe/meta.yaml (hunk 1 above)
Comment (jhamrick) [reply recipe/meta.yaml]: Good catch!

**Comment by jhamrick** [comment]
> please could you also re-render this with conda smithy 2.3.2

Oh, yes. I tried doing that but must have forgotten to update conda smithy...

**Comment by jhamrick** [comment]
Ok, done @lgpage !

**Review by lgpage — APPROVED** [review]

**Pull request merged**